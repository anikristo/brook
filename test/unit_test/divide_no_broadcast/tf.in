2 9
4 5 6 2 6 -3 -5 4 -22
5 7 9 6 11 3 2 12 -13