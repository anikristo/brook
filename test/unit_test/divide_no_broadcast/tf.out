2 9
4.2579 -1.53525 6.36555 -4.27056 0.0513335 1.04549 0.471493 -0.454046 -68.606
19.7462 -29.7486 -1.32914 -0.864905 3.33643 -0.591703 -0.357392 -0.482282 16.714
